sample/kotlin/src/main/kotlin/com/datadog/android/sample/SampleApplication.kt

```diff
@@ -129,6 +131,11 @@ class SampleApplication : Application() {
                     SampleNavigationPredicate()
                 )
             )
+            .setRumViewEventMapper(object : ViewEventMapper {
```

Comment:
I'm all for adding stuff in the sample app to showcase features of the SDK, but here this does the same thing as the `NoOpViewEventMapper`. Maybe we could redact a field here (e.g; the view.url or something)?

Reply:
Yup...my intention was to test this...I think I will just drop this and prepare a special page in the sample app for this mappers in the task where we will have to align the error/resource count if the events were dropped.